README.md

```diff
@@ -65,7 +63,7 @@ jobs:
 
     steps:
       - name: Run action
-        uses: steffen-karlsson/stale-discussions@v1
+        uses: steffen-karlsson/stalesweeper@v1
```

Comment:
Fixed it here and everywhere else, otherwise it was still pointing to the old name